### PR TITLE
Update: Classifying Metrics Based on Request or Response

### PR DESCRIPTION
**Pull request by craigbox**
"In this release, Request Classification has been promoted to Beta."

[ ] Configuration Infrastructure
[X] Docs
[X] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by craigbox** [comment]
Please also see if the things that say '1.8" on this page need to now say "1.9".

**Comment by ericvn** [comment]
@istio/wg-policies-and-telemetry-maintainers See @craigbox comment above.

**Comment by rvennam** [comment]
The EnvoyFIlters are scoped to 1.8. These and the examples need to be updated to the right version... istio/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.9.yaml

**Comment by craigbox** [comment]
> The EnvoyFIlters are scoped to 1.8. These and the examples need to be updated to the right version... istio/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.9.yaml

done

**Comment by linsun** [comment]
Hi Craig, can you link to the beta checklist url in this PR?  See enhancements repo for examples

**Comment by craigbox** [comment]
> Hi Craig, can you link to the beta checklist url in this PR? See enhancements repo for examples

I don't see an enhancement checklist for this.  I'm only going on the fact the blog post said that it had gone to Beta, but the docs hadn't.  Who tracks this sort of thing?  /cc @jacob-delgado as release manager to triage

**Review by rvennam — APPROVED** [review]

**Review by rvennam — CHANGES_REQUESTED** [review]
There are still many references to 1.8 in this doc. Do you want to not tackle those in this PR?

image

File: content/en/docs/tasks/observability/metrics/classify-metrics/index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ For more information, see the
 
 Istio uses the Envoy proxy to generate metrics and provides its configuration in
 the `EnvoyFilter` at
-[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml).
+[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.9.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.9.yaml).
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/classify-metrics/index.md:38]: Should we use {{< istio_version >}} so this changes with release?

Comment (craigbox) [reply content/en/docs/tasks/observability/metrics/classify-metrics/index.md:38]: > There are still many references to 1.8 in this doc. Do you want to not tackle those in this PR?
> 
> image

Adding @richardwxn as he seems to have updated s/1.6/1.8/ in #8339 and may have opinions.

(All **I** wanted to do was remove the Experimental!)

Comment (richardwxn) [reply content/en/docs/tasks/observability/metrics/classify-metrics/index.md:38]: @craigbox we have to update all the references to 1.8 to 1.9 in this page

**Comment by craigbox** [comment]
Closing in favour of #9772.

**Pull request closed without merging**